drivers/serial/uart_stm32.c

```diff
@@ -1615,7 +1615,8 @@ static int uart_stm32_init(const struct device *dev)
 		return err;
 	}
 
-	LL_USART_Disable(config->usart);
+	/* Reset UART to default state using RCC */
+	LL_USART_DeInit(config->usart);
```

Comment:
These are 2 different things. Removing `LL_USART_Disable` may be valid. But this should be done in a follow up commit.

Reply:
Removing `LL_USART_Disable` is valid because after `LL_USART_DeInit` all UART registers have default values. Moved removing `LL_USART_Disable` to separate commit.